### PR TITLE
Include a clause for properly finding Multi-day on date search

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#121868](http://central.tri.be/issues/121868)
---

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Day.php:201](https://github.com/moderntribe/the-events-calendar/blob/8c93c5a332f350b76792f5efff5b77b8da478ed0/src/Tribe/Template/Day.php#L201)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Day.php:201](https://github.com/moderntribe/the-events-calendar/blob/c574b7eb24aac5bebc5173319d766259b54c482d/src/Tribe/Template/Day.php#L201)

> via jenkins/codesniffer-phpcs

**Comment by bordoni** [comment]
The Tests that involve API seem to be failing for some reason, on all plugins.

File: src/Tribe/Template/Day.php [hunk 1]
```diff
@@ -185,14 +184,36 @@ public function ajax_response() {
...
 				}
 
-				/** @var WP_Query query */
-				$query = tribe_get_events( $args, true );
+				$event_date = tribe_get_request_var( 'eventDate', '' );
+				if ( empty( $event_date ) ) {
+					$event_date = date( 'Y-m-d', current_time( 'timestamp' ) );
+				}
+
+				$args['posts_per_page'] = - 1; // show ALL day posts
```
Comment (mitogh) [inline src/Tribe/Template/Day.php]: I think this unary operator should be close to the number to make it more clear that is `-1` instead of `- 1`

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Day.php:201](https://github.com/moderntribe/the-events-calendar/blob/8d8bd721740d8822539ec108eae57dc38cce795d/src/Tribe/Template/Day.php#L201)

> via jenkins/codesniffer-phpcs

**Pull request merged**